docs/getstarted.md

```diff
@@ -233,6 +231,55 @@ trainer = Trainer(
 trainer.train(nepoch=50, batch_size = 64)
 ```
 
+### Results export and visualization
+
+The user can specify a deeprankcore exporter or a custom one in `output_exporters` parameter of the Trainer class, together with the path where to save the results. Exporters are used for storing predictions information collected later on during training and testing. Example:
+
+```python
+from deeprankcore.trainer import Trainer
+from deeprankcore.ginet import GINet
+from deeprankcore.utils.exporters import HDF5OutputExporter
+
+trainer = Trainer(
+    GINet,
+    dataset_train,
+    dataset_val,
+    dataset_test,
+    output_exporters = [HDF5OutputExporter("<output_folder_path>")]
+)
+```
+
+By default, the Trainer class creates the folder `./output` and uses `HDF5OutputExporter`. In the latter case, results are saved in `output_exporter.hdf5` both during the training (`.train()`) and during the testing (`.test()`) phases. `output_exporter.hdf5` contains Groups which refer to each phase, e.g. `training` and `testing` if both were run, only one of them otherwise. Training here includes validation as well. The HDF5 file can then be read as a [Pandas Dataframe](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html):
```

Comment:
```suggestion
By default, the Trainer class creates the folder `./output` and uses `HDF5OutputExporter`. In the latter case, results are saved in `output_exporter.hdf5` both during the training (`.train()`) and during the testing (`.test()`) phases. `output_exporter.hdf5` contains Groups which refer to each phase, e.g. `training` and `testing` if both are run, only one of them otherwise. Training phase includes validation results as well. The HDF5 file can then be read as a [Pandas Dataframe](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html):
```